src/bootstrap/cloud/terraform/address.tf

```diff
@@ -2,13 +2,13 @@ resource "google_compute_address" "cloud_robotics" {
   name       = "cloud-robotics"
   project    = data.google_project.project.project_id
   region     = var.region
-  depends_on = [google_project_service.compute]
+  depends_on = [google_project_service.project-services["compute"]]
```

Comment:
Should these names be fully-qualified? How does it map "compute" to "compute.googleapis.com" now that "compute" no longer appears literally in project.tf?

Reply:
You are right. Interestingly terraform did not complain, but the full key is clearly visible from the plan output:
```
google_project_service.project-services["endpoints.googleapis.com"]: Refreshing state... [id=robco-ensonic/endpoints.googleapis.com]
google_project_service.project-services["dns.googleapis.com"]: Refreshing state... [id=robco-ensonic/dns.googleapis.com]
google_project_service.project-services["compute.googleapis.com"]: Refreshing state... [id=robco-ensonic/compute.googleapis.com]
...
```